GearSpawner/Patches.cs

```diff
@@ -1,34 +1,46 @@
 ﻿using HarmonyLib;
 using Il2Cpp;
+using UnityEngine;
+using UnityEngine.AddressableAssets;
 
 namespace GearSpawner;
 
 internal static class Patches
 {
-	[HarmonyPatch(typeof(LootTable), nameof(LootTable.GetPrefab))]
-	internal static class LootTable_GetPrefab
+	// patch the look tables as they initialize
+	[HarmonyPatch(typeof(LootTableData), nameof(LootTableData.Initialize))]
+	internal static class LootTableData_Awake
 	{
-		private static void Prefix(LootTable __instance)
+		private static void Postfix(LootTableData __instance)
 		{
-			LootTableManager.ConfigureLootTable(__instance);
+				bool changed = LootTableManager.ConfigureLootTableData(__instance);
 		}
 	}
 
-	[HarmonyPatch(typeof(LootTable), nameof(LootTable.GetRandomGearPrefab))]
-	internal static class LootTable_GetRandomGearPrefab
+	//patch the scenes for loose items as they load
+	[HarmonyPatch(typeof(QualitySettingsManager), nameof(QualitySettingsManager.ApplyCurrentQualitySettings))]//Exists
+	internal static class GameManager_ApplyCurrentQualitySettings
 	{
-		private static void Prefix(LootTable __instance)
+		private static void Prefix()
 		{
-			LootTableManager.ConfigureLootTable(__instance);
+			GearSpawnManager.PrepareScene();
 		}
 	}
 
-	[HarmonyPatch(typeof(GameManager), nameof(GameManager.SetAudioModeForLoadedScene))]//Exists
-	internal static class GameManager_SetAudioModeForLoadedScene
+	// this patch is required to allow non md5 asset reference names to be "valid"
+	[HarmonyPatch(typeof(AssetReference), nameof(AssetReference.RuntimeKeyIsValid))]//Exists
```

Comment:
Remove //exists everywhere in the mod. It's a remnant of when I was first porting Wulf's code.